tests/migrations/test_runner_individual.py

```diff
@@ -39,67 +39,71 @@ def teardown_function() -> None:
 
 def test_transactions_compatibility() -> None:
     cluster = get_cluster(StorageSetKey.TRANSACTIONS)
-    connection = cluster.get_query_connection(ClickhouseClientSettings.MIGRATE)
 
-    def get_sampling_key() -> Any:
-        database = cluster.get_database()
-        ((sampling_key,),) = perform_select_query(
-            ["sampling_key"],
-            "system.tables",
-            {"name": "transactions_local", "database": str(database)},
-            None,
-            connection,
+    if cluster.is_single_node():
+        connection = cluster.get_query_connection(ClickhouseClientSettings.MIGRATE)
```

Comment:
It's a bit more readable to just early return if it's not a single node and not increase the indentation of the whole test which is very long, i.e.
```
if cluster.is_single_node() is False:
    return
```
Please also add a comment about why we don't want to run the test in distributed mode.

Reply:
Ok!